src/RBACHandler.ts

```diff
@@ -48,11 +48,14 @@ export class RBACHandler implements Authorization {
         return this.isAllowed(groups, request.operation, request.resourceType);
     }
 
+    // eslint-disable-next-line class-methods-use-this
+    isAllowedToAccessBulkDataJob(requesterId: string, jobOwnerId: string): boolean {
+        return requesterId === jobOwnerId;

```

Comment:
We will need to add this method to the Authorization interface and use a request object instead of inline params